ambari/datadog_checks/ambari/ambari.py

```diff
@@ -38,40 +36,30 @@ def check(self, instance):
 
         clusters = self.get_clusters(clusters_endpoint)
         self.get_hosts_metrics(clusters, server, port, tags)
-        
 
-        
     def get_clusters(self, url):
         cluster_list = []
-        try:
-            r = self.http.get(url)
-            r.raise_for_status()
-        except:
```

Comment:
capturing an exception to raise it again does not make much sense

Reply:
I was using this as a skeleton framework for exceptions (although I agree it doesn't really make sense as is).  I'll take it out for now and can add back in when I actually write the exceptions.